uat/testing-features/src/main/resources/local-store/recipes/client_python_paho.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+---
+RecipeFormatVersion: '2020-01-25'
+ComponentName: aws.greengrass.client.Mqtt5PythonPahoClient
+ComponentVersion: '1.0.0'
+ComponentDescription: MQTT 5.0/3.1.1 Python Client powered by Eclipse Paho
+ComponentPublisher: Amazon
+ComponentConfiguration:
+  DefaultConfiguration:
+    # agentId should be the same as ComponentName
+    agentId: aws.greengrass.client.Mqtt5PythonPahoClient
+    controlAddresses: 127.0.0.1
+    controlPort: 47619
+Manifests:
+  - Artifacts:
+      - URI: classpath:/local-store/artifacts/client-python-paho
+        Permission:
+          Read: ALL
+          Execute: ALL
```

Comment:
OWNER to avoid cases when anybody can run client

Reply:
fixed in b3d3712